packages/wonder-blocks-form/src/components/text-area.tsx

```diff
@@ -34,6 +40,12 @@ type TextAreaProps = AriaProps & {
      * Custom styles for the text area.
      */
     style?: StyleType;
+    /**
+     * Custom styles for the root node of the component.
+     * If possible, try to use this prop carefully and use the `style` prop
+     * instead.
+     */
+    rootStyle?: StyleType;
```

Comment:
Let me know if you have any feedback on this approach! I saw this approach with the cell props:

https://github.com/Khan/wonder-blocks/blob/490ddd6ed93e5b8c0b4640756f164c48b563ba56/packages/wonder-blocks-cell/src/util/types.ts#L102-L108